src/scripts/quests/QuestLineHelper.ts

```diff
@@ -3,6 +3,7 @@
  */
 class QuestLineHelper {
 
+    //Kanto QuestLines
```

Comment:
Space between // and Kanto, to follow what we are doing other places

Reply:
KK I fix